### PR TITLE
WINDUPRULE-220

### PR DESCRIPTION
**Pull request by OndraZizka**

### CODE REVIEW

File: rules-reviewed/eap7/eap6/tests/eap6.windup.test.xml [hunk 1]
```diff
@@ -129,7 +129,7 @@
             <rule id="eap6-11000-test">
                 <when>
                     <not>
-                         <hint-exists message="Value of `FaceletContext.FACELET_CONTEXT_KEY` has changed"/>
+                         <hint-exists message="The alue of JSF constant `FaceletContext.FACELET_CONTEXT_KEY` has changed"/>
```
Comment (mareknovotny) [inline rules-reviewed/eap7/eap6/tests/eap6.windup.test.xml]: typo in "alue" instead of "value"?

Comment (OndraZizka) [inline rules-reviewed/eap7/eap6/tests/eap6.windup.test.xml]: Damn how could I miss that, thx

Comment (OndraZizka) [inline rules-reviewed/eap7/eap6/tests/eap6.windup.test.xml]: Fixed

**Comment by jsight** [comment]
+1

**Pull request merged**